ask-sdk-dynamodb-persistence-adapter/lib/attributes/persistence/PartitionKeyGenerators.ts

```diff
@@ -62,4 +62,21 @@ export const PartitionKeyGenerators = {
 
         return requestEnvelope.context.System.device.deviceId;
     },
+
+    /**
+     * Gets attributes id using person id.
```

Comment:
This misses out the fallback information of returning user id if person id is not present. Can you add that as well?